src/internal/require/require.go

```diff
@@ -1,9 +1,9 @@
 // Package require implements test assertions.
 package require
 
+//nolint:printf
```

Comment:
If you fixed the lint, can this line be removed now?

Reply:
It in fact didn't do anything.

Reply:
Removed.